CONTRIBUTING.md

```diff
@@ -96,8 +96,9 @@ In this scenario, it’s possible that you found a bug or enhancement that we ha
 	2. Tests - All tests must pass before a merge of a pull request
 	3. Lint - All code must pass lint checks before a merge of a pull request
 13. Please never force push when a PR review has already started (because this messes with the PR review history)
-14. Upon submission of a PR, please include a numbered list of explicit testing steps for each platform (Web, Desktop, iOS, and Android) to confirm the fix works as expected and there are no regressions.
-15. Please add a screenshot of the app running on each platform (Web, Desktop, iOS, Android, Mobile Web). If you are unable to build to iOS/OSX due to using a Windows machine, please let the reviewers know so they can double check that platform themselves.
+14. Please, pay attention to the pull request template, especially to how we require issues to be linked with the PR. We have processes in place to help us manage the open source issues and to pay contributors on time. If Github keywords such as `fixes` are used to link issue to PR, this will automatically close the related issue once the PR is merged and break our flow. Please, do not use these keywords and use format stated in the PR template.
+15. Upon submission of a PR, please include a numbered list of explicit testing steps for each platform (Web, Desktop, iOS, and Android) to confirm the fix works as expected and there are no regressions.
```

Comment:
Hmm I'm a bit surprised we don't have `mWeb` here but we do on the next step. Maybe we can remove the list of platforms in step 16 since it already exists in this step? Or, at least keep both lists of platforms consistent :D

```suggestion
15. Upon submission of a PR, please include a numbered list of explicit testing steps for each platform (Web, Desktop, iOS, Android, and Mobile Web) to confirm the fix works as expected and there are no regressions.
```